internal/controller/components/trustyai/trustyai_controller_actions.go

```diff
@@ -82,6 +82,22 @@ func createConfigMap(ctx context.Context, rr *odhtypes.ReconciliationRequest) er
 		return fmt.Errorf("resource instance %v is not a componentApi.TrustyAI)", rr.Instance)
 	}
 
+	// Extract values and convert string to boolean for configmap
+	permitCodeExecutionStr := trustyai.Spec.Eval.LMEval.PermitCodeExecution
+	permitOnlineStr := trustyai.Spec.Eval.LMEval.PermitOnline
+
+	// Default to "deny" if empty
+	if permitCodeExecutionStr == "" {
+		permitCodeExecutionStr = EvalPermissionDeny
+	}
+	if permitOnlineStr == "" {
+		permitOnlineStr = EvalPermissionDeny
+	}
+
+	// Convert to boolean for configmap
+	permitCodeExecution := permitCodeExecutionStr == EvalPermissionAllow
+	permitOnline := permitOnlineStr == EvalPermissionAllow
```

Comment:
this could be probably be simplified like:

```go
permitCodeExecution := trustyai.Spec.Eval.LMEval.PermitCodeExecution == EvalPermissionAllow
permitOnline := trustyai.Spec.Eval.LMEval.PermitOnline == EvalPermissionAllow
```

Unless I missed something, this would work correctly because:
- If the field is empty (""), comparing it to `EvalPermissionAllow` returns false (which is the desired "deny" behavior)
- If the field is `EvalPermissionAllow`, it returns `true`
- If the field is `EvalPermissionDeny`, it returns `false`